src/app/page.tsx

```diff
@@ -1,16 +1,50 @@
 import Image from "next/image";
+import React from 'react';
+
 
 export default function Home() {
+
+
+ const cardsData =[
+  {image: '', title: "Card1", content: ""  },
+  {image: '', title: "Card2", content: ""  },
+  {image: '', title: "Card3", content: ""  },
+  {image: '', title: "Card4", content: ""  },
+  {image: '', title: "Card5", content: ""  },
+  {image: '', title: "Card6", content: ""  },
+ ]
+
+ //React.FC creates reusable components in React applications!!! you pass the props and the info in it as param
+
+ const CardComponent : React.FC<{ className: string }> = ({className}) => {
+  return(
+    <div>
+      {cardsData.map((card) => ( // .map creates a new array
```

Comment:
If the component iscalled card it should have the implementation that handles only one card, instead of having it handle list of cards. Prefer handling one card inside CardComponent and using the map where you intend to use the actual component. All the information should be passed by props.

Prefer:
- card implements only one card
- data is passed by props
- biggest component controls the data(Page)
```tsx
const Card:FC<Props> = ({a,b,c}) => {
  return <div>
     <h1>{a}</h1>
     <h1>{b}</h1>
     <h1>{c}</h1>
  </div>
}

const Page:FC<Props> = () => {
    const data = [{...},{...},{...}];
    
    return <div>
      {data.map(dataItem => (
         <Card 
             a={dataItem.a}
             b={dataItem.b}
             c={dataItem.c}
         />
      )}
    </div>;
}
```



instead of:

```tsx
const Card:FC<Props> = ({a,b,c}) => {

 const data = [{...},{...},{...}];
    
    return <div>
      {data.map(dataItem => (
          <h1>{dataItem.a}</h1>
          <h1>{dataItem.b}</h1>
          <h1>{dataItem.c}</h1>
      )}
    </div>;
}

const Page:FC<Props> = () => {    
    return <div>
      <Card />
    </div>;
}
```